jobs/invitations/RemoveExpiredInvitationsJob.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We've decided not to use `strict_types`; please remove these throughout the PRs.

Reply:
I think I removed it from everywhere in the PRs